commands/moderation/notes.js

```diff
@@ -62,11 +62,11 @@ function notesLog(msg, targetUser, notes) {
       )
       .setColor(embedFlair[Math.floor(Math.random() * embedFlair.length)])
       .addField(`Total`, `${totalNotes}`)
-      .addField(`Latest Notes: `, listOfNotes)
+      .addField(`Latest Notes: `, listOfNotes.join('\n'))
```

Comment:
This is interesting, did they change it so you can't pass in a list here anymore? That's good to know.